src/deviceApiCalls/__generated__/validators-ts.d.ts

```diff
@@ -337,6 +337,15 @@ export interface SelectedDetailParams {
   configType: string;
 }
 
+// sendJSPixel.params.json
+
+/**
+ * Send pixels data to be fired from the native layer
+ */
+export interface SendJSPixelParams {
+  pixelName: "autofill.identity.count";
```

Comment:
Is it okay if this is changed to `autofill_identity_count`? The native layer wants the components in its pixel names to be separated by underscore, [like this](https://github.com/more-duckduckgo-org/macos-browser/blob/develop/DuckDuckGo/Statistics/PixelEvent.swift#L208).

Reply:
I think we can also drop the `count` here too since that's specific to the Grafana syntax - so the final pixel will be `autofill_identity`, which matches the others we have (`m_mac_autofill_password` and `m_mac_autofill_card`, where the macOS browser will append the `m_mac` part, and I'll do the equivalent for iOS).